api/Dockerfile

```diff
@@ -48,7 +48,9 @@ WORKDIR /app
 COPY nginx.conf.template /etc/nginx/templates/openverse-api.conf.template
 COPY /static /app/static
 
-ENV NGINX_ENVSUBST_FILTER="DJANGO_UPSTREAM_URL"
+# Only environment variables with this prefix will be available in the template
+ENV NGINX_ENVSUBST_FILTER="DJANGO_NGINX_"
+ENV DJANGO_NGINX_ENVIRONMENT="local"
```

Comment:
We'll need to be sure to pass this environment variable in the ECS configuration as well, once we get there. That at least is constant for the environment, so not an issue.

Reply:
Yes, same with the `DJANGO_NGINX_GIT_REVISION`!